vuepress/docs/ecr/tutorials/create-ecr-bundle-from-git.md

```diff
@@ -0,0 +1,107 @@
+# Creating an Entando Component Repository (ECR) bundle using git repository
+
+## Purpose
+
+Generate a simple Entando Component Repository (ECR) bundle shareable in an Entando 6 environment using git repository
+
+## Requirements
+
+You can create the bundle using you favorite code editor. To share the bundle you will need:
+1. NodeJS, `npm`, and `git`
+1. A git repository where bundle with descriptor.yaml is stored
```

Comment:
I think this should be "A git repository where to store a demo bundle", otherwise is not clear why you need to create a bundle later if you already have in the requirements a bundle with a descriptor.yaml in git